scripts/testSendGridTemplates.js

```diff
@@ -0,0 +1,23 @@
+// Script to test every (dynamic) SendGrid email template with SendGrid email API
+// The env email address should recieve every template type defined in ENV (sent from itself)
+var Mailer = require('../app/server/services/sendgrid_email');
+
+var email = process.env.EMAIL_ADDRESS;
```

Comment:
If you're putting this in the scripts folder, then you'll need a copy of the .env to run it from this folder.
Youll need to add require('dotenv').load(); in order for the environment var to be seen. if you're running it stand alone.